sqlmesh/core/dialect.py

```diff
@@ -645,3 +646,20 @@ def extract_columns_to_types(query: exp.Subqueryable) -> t.Dict[str, exp.DataTyp
         expression.output_name: expression.type or exp.DataType.build("unknown")
         for expression in query.selects
     }
+
+
+def find_tables(expression: exp.Expression, dialect: DialectType = None) -> t.Set[str]:
+    """Find all tables referenced in a query.
+
+    Args:
+        expressions: The list of expressions to find tables for.
```

Comment:
Haha @izeigerman sorry for nitpicking on this, there's still a trailing `s` in the docstring, i.e. `expression*s*` instead of `expression` (it doesn't match the actual parameter name).